src/mongo_logger.ts

```diff
@@ -738,6 +750,37 @@ export class MongoLogger {
     return compareSeverity(severity, this.componentSeverities[component]) <= 0;
   }
 
+  turnOffSeverities() {
+    for (const key of Object.keys(MongoLoggableComponent)) {
+      this.componentSeverities[key as MongoLoggableComponent] = SeverityLevel.OFF;
+    }
+  }
+
+  private logWriteFailureHandler(error: Error) {
+    try {
+      if ((this.logDestination as any)?.streamName === 'stderr') {
+        throw MongoError;
```

Comment:
This was a little strange at first, but I follow, you are jumping down to the catch block. Since the error is going to be discarded anyway can you leave something descriptive: `throw new Error('...')` where ... is something about skipping over the stderr log. And I use `Error` here because that will also stand out to anyone reading this

Alternatively, switch this to an early return style if stmt, it incurs less scrutiny when others read this.

Reply:
I've changed it to an early return statement for clarity.